release-notes.md

```diff
@@ -354,6 +360,50 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v44.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v44.0. The following user visible changes have been made:
+
+### Configurable IOMMU Address Width
+
+The `iommu_address_width` option has been added to `--platform` to allow users
```

Comment:
This is very specifically `virtio-iommu` it's under platform because we don't have an `iommu` top level config.

Reply:
Updated.